COSC481W-2024Fall/Ript-Fitness
# Workouts page

Description:
This pull request is the initial version of the workouts page. The workouts page is the default screen on the workouts tab in which one can select where they would like to head from there. Either to go to My workouts, add workout, ript workouts, start workout. 
Key Changes:

- implemented a new tab for the page

- has 4 buttons with the options displayed

- added icons for each button

- added an animation for when the buttons are clicked to show they have been pressed

- changed colors and sizes to ensure the UI looks clean 
What’s Not Completed:
- currently the onPress is a filler to the apiscreen need to change this to go to the associated pages
- need to disscuss with group about finalizing sizing and potentially adding a another button to fill the white space

Testing Instructions:
 
- navigate to the workouts tab
- all 4 buttons should be ther
- click on any button for now they should take you to the apiscreen
- verify the buttons properly work with onpress 
- use expo go with npx expo start

the link to issue 6
https://github.com/COSC481W-2024Fall/Ript-Fitness/issues/6
